sys/fuchsia/fidl_process.txt

```diff
@@ -0,0 +1,10 @@
+include <zircon/syscalls.h>
+
+resource zx_chan_fuchsia_process_Launcher_local[zx_chan]
```

Comment:
I know it was me who proposed "local"/"remote". I did not know what are common names for this, and if they exist at all. But now that I read the spec:
https://fuchsia.googlesource.com/zircon/+/master/docs/fidl/wire-format/index.md
I think it's better to stick to the established terminology of "client"/"server" (we will fdio_service_connect the server one, right?).